s3cloudfront/policies.tf

```diff
@@ -0,0 +1,20 @@
+# Create IAM and Bucket policies
+data "aws_iam_policy_document" "s3_pol" {
+  statement {
+    actions = [
+      "s3:PutObject",
```

Comment:
Can we just limit this to `s3:GetObject`? I don't think we will allow for PUT/DELETE of objects through the CDN. This should simplify access control.